src/net/sourceforge/kolmafia/persistence/DailyLimitDatabase.java

```diff
@@ -128,44 +138,55 @@ public int getId() {
     }
 
     public int increment() {
+      return increment(1);
+    }
+
+    public int increment(final int delta) {
       if (getMax() == 1 && Preferences.getDefault(this.uses).equals("false")) {
         Preferences.setBoolean(this.uses, true);
         return 1;
       }
 
-      return Preferences.increment(this.uses, 1, getMax(), false);
+      if (isTome()) {
+        Preferences.increment("tomeSummons", delta, getMax(), false);
+      }
+
+      return Preferences.increment(this.uses, delta, getMax(), false);
     }
   }
 
   static {
-    for (DailyLimitType type : DailyLimitType.values()) {
-      DailyLimitDatabase.tagToDailyLimitType.put(type.toString(), type);
+    for (var type : DailyLimitType.values()) {
+      tagToDailyLimitType.put(type.toString(), type);
     }
 
-    DailyLimitDatabase.reset();
+    tagToDailyLimitType.put("tome", DailyLimitType.CAST);
+
+    reset();
   }
 
   public static void reset() {
     boolean error = false;
-    try (BufferedReader reader =
-        FileUtilities.getVersionedReader("dailylimits.txt", KoLConstants.DAILYLIMITS_VERSION)) {
-      String[] data;
-
-      while ((data = FileUtilities.readData(reader)) != null) {
-        if (data.length >= 2) {
-          String tag = data[0];
-
-          DailyLimitType type = DailyLimitDatabase.tagToDailyLimitType.get(tag.toLowerCase());
-          DailyLimit dailyLimit = parseDailyLimit(type, data);
-          if (dailyLimit == null) {
-            RequestLogger.printLine("Daily Limit: " + data[0] + " " + data[1] + " is bogus");
-            error = true;
-          }
+
+    BufferedReader reader =
```

Comment:
You've dropped a `try (...)` here.

Reply:
On purpose! Unless I'm mistaken this exception is not thrown, its caught in FileUtilities.readLine

Reply:
it's not about the exception, it's about closing the reader.

Reply:
Ah so we need the try but not the catch. Will fix this later, I'm off to a concert!

Reply:
Performing or listening?

Reply:
you need the catch because the `close` method, by Java's exceptions, can throw. Realistically I don't think it's going to but that's Java for you.